src/shared/lib/oql/AccessorsForOqlFilter.ts

```diff
@@ -107,7 +115,14 @@ export default class AccessorsForOqlFilter {
         );
     }
 
-    public gene(d: Mutation) {
+    private isMutation(d: Datum): d is Mutation {
+        return (
+            this.molecularAlterationType(d.molecularProfileId) ===
+            'MUTATION_EXTENDED'
```

Comment:
Don't we have this in enum somewhere?

Reply:
yes, good catch, I was just copy pasting code here